scripts/security/certs-create.sh

```diff
@@ -11,7 +11,12 @@ rm -f *.crt *.csr *_creds *.jks *.srl *.key *.pem *.der *.p12 *.log
 # Generate CA key
 openssl req -new -x509 -keyout snakeoil-ca-1.key -out snakeoil-ca-1.crt -days 365 -subj '/CN=ca1.test.confluentdemo.io/OU=TEST/O=CONFLUENT/L=PaloAlto/ST=Ca/C=US' -passin pass:confluent -passout pass:confluent
 
-users=(kafka1 kafka2 client schemaregistry restproxy connect connectorSA controlcenter ksqlDBserver ksqlDBUser appSA badapp clientListen zookeeper mds)
+# we no longer generating a cert for ksqlDB Server - it shares a cert with Control Center
```

Comment:
1. Let's clean up the comment, e.g. `ksqlDB (ksqldbserver) and Control Center (controlcenter) share a certificate`.

2. Can you add to the comment to explain *why* they can't have distinct certificates in this demo?  This will help us remember in 1 year ;)